changelog.d/1366.change.rst

```diff
@@ -1 +1,2 @@
 In package_index, fixed handling of encoded entities in URLs.
+Scripts which have unicode content are now sopported
```

Comment:
This should be in its own changelog file, `changelog.d/1389.change.rst`

Reply:
Also `s/sopported/supported`

Reply:
fixed